### ISSUE TITLE
Include HTTP2 Proxy Boolean on App Service in REST API

### ISSUE THREAD

**Issue by jameswoodley** [Azure/azure-rest-api-specs#22876]
In order to use this setting in tools like Terraform or Pulumi, we need the HTTP2 Proxy included in the REST Api.

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/52610563/222388393-3f01d357-104c-4d34-8c79-5adb0083695e.png">

**Comment by navba-MSFT** [comment]
@jameswoodley Apologies for the late reply. Thanks for reaching out to us and reporting this issue. We are looking into this issue and we will provide an update.

**Comment by navba-MSFT** [comment]
@jameswoodley  I could see that the Azure Portal invoked the below 2 APIs while setting the `http20Enabled `and `http20ProxyFlag` properties.

[Web Apps - Create Or Update ](https://learn.microsoft.com/en-us/rest/api/appservice/web-apps/create-or-update)
[Web Apps - Update Slot Configuration Names](https://learn.microsoft.com/en-us/rest/api/appservice/web-apps/update-slot-configuration-names)

>properties.siteConfig.http20Enabled=true
>properties.siteConfig.http20ProxyFlag=1

Please try the same and let me know if that helps.

**Comment by navba-MSFT** [comment]
@jameswoodley I wanted to do quick follow-up to check if you had a chance to look at my above comment. Please let us know if you had any updates on this. Awaiting your reply.

**Comment by jameswoodley** [comment]
@navba-MSFT we are checking on the Terraform end, I'll revert back once I know more

**Comment by navba-MSFT** [comment]
@jameswoodley Thanks for getting back. Please let us know if you had any update.

**Comment by navba-MSFT** [comment]
@jameswoodley I wanted to do quick follow-up to check if you had any updates on this. Awaiting your reply.

**Comment by navba-MSFT** [comment]
@jameswoodley The action is pending on you to get back to us for [my above suggesstion](https://github.com/Azure/azure-rest-api-specs/issues/22876#issuecomment-1460100547). Please feel free to reopen this thread. We would be happy to help.

**Comment by xiaxyi** [comment]
Hi @navba-MSFT , Thanks for helping with this issue, I can't see the property http20ProxyFlag  in the app service 2021-03-01 api https://github.com/Azure/azure-rest-api-specs/blob/main/specification/web/resource-manager/Microsoft.Web/stable/2021-03-01/WebApps.json

can you double confirm if you added it to the swagger? Thanks

**Comment by jameswoodley** [comment]
@navba-MSFT update here 👍🏻

**Comment by jameswoodley** [comment]
@navba-MSFT I don't seem to be able to reopen the issue, but update here 👍🏻

**Comment by algompluecker** [comment]
@jameswoodley is there any update on this?

Thanks in advance!

**Comment by jameswoodley** [comment]
> @jameswoodley is there any update on this?
> 
> Thanks in advance!

We are awaiting a reply to our last message, see above

**Comment by jameswoodley** [comment]
@navba-MSFT any updates for us?

**Comment by jameswoodley** [comment]
@navba-MSFT still awaiting a response.. it's been many months. I will open a new issue if no response here within the week

cc @JackTn
